spec/requests/citizens/restrictions_spec.rb

```diff
@@ -14,54 +14,64 @@
     it 'returns http success' do
       expect(response).to have_http_status(:ok)
     end
-
-    it 'displays the restriction name labels' do
-      restrictions.map(&:label_name).each do |label|
-        expect(unescaped_response_body).to include(label)
-      end
-    end
   end
 
-  describe 'POST /citizens/restrictions' do
-    let(:restriction_ids) { restrictions.pluck(:id) }
+  describe 'PATCH /citizens/restrictions' do
     let(:params) do
       {
-        continue_button: 'Continue',
         legal_aid_application: {
-          restriction_ids: restriction_ids
+          has_restrictions: has_restrictions,
+          restrictions_details: restrictions_details
         }
       }
     end
-    subject { post citizens_restrictions_path, params: params }
+    let(:restrictions_details) { Faker::Lorem.paragraph }
+    let(:has_restrictions) { true }
```

Comment:
`'true'` better matches what the real param to the controller is
```suggestion
    let(:has_restrictions) { 'true' }
```